packages/e3kit-browser/README.md

```diff
@@ -44,6 +44,20 @@ This package is **ONLY** for browsers. Use the following packages on other platf
       const EThree = window.E3kit.EThree;
   </script>
   ```
+  
+## Encrypt & decrypt large files
+
+If you need to encrypt & decrypt large files with the best speed/browser perfomance ratio, see the `encryptFile` and `decryptFile` methods:
+- https://virgilsecurity.github.io/virgil-e3kit-js/classes/ethree.html#encryptfile
+- https://virgilsecurity.github.io/virgil-e3kit-js/classes/ethree.html#decryptfile
+
+Both methods take an instance of `File` class as input instead of binary `ArrayBuffer`.
+The files are encrypted in small chunks, so it doesn't block the main thread and it returns an encrypted instance of `File`. The chunk size by default is 64kb which produces the best speed/browser performance ratio, but it can be changed. Larger chunk size speeds up encryption but can cause browser lags.
+
+Simple demo based on the methods above: https://virgilsecurity.github.io/virgil-e3kit-js/encryptFile.html
```

Comment:
Right now we don't have that demo published on GitHub Pages, so it's better to remove it.
We have its source code available [here](https://github.com/VirgilSecurity/virgil-e3kit-js/blob/master/examples/encryptFile.html).